files/en-us/mdn/contribute/markdown_in_mdn/index.html

```diff
@@ -113,7 +113,108 @@ <h3>Example code blocks</h3>
 ```
 </pre>
 
-<h3>Notes and warnings</h3>
+<h3>Notes, warnings, and callouts</h3>
+
+<p>Sometimes writers want to call special attention to some piece of content. To do this, they will use a GFM blockquote with a special first paragraph. There are three types of these: notes, warnings, and callouts:</p>
+
+<h4>Notes</h4>
+
+<p>To write a note, an author creates a GFM blockquote whose first paragraph starts with "**Note:**":</p>
+
+<pre>
+&gt; **Note:** This is how you write a note.
+&gt;
+&gt; It can have multiple lines.
+</pre>
+
+<p>This will produce the following HTML:</p>
+
+<pre class="brush: html">
+&lt;div class="notecard note"&gt;
+  &lt;p&gt;&lt;strong&gt;Note:&lt;/strong&gt; This is how you write a note.&lt;/p&gt;
+  &lt;p&gt;It can have multiple paragraphs.&lt;/p&gt;
+&lt;/div&gt;
+</pre>
+
+<p>This HTML will be rendered as a highlighted box, like:</p>
+
+<div class="notecard note">
+  <p><strong>Note:</strong> This is how you write a note.</p>
+  <p>It can have multiple paragraphs.</p>
+</div>
+
+<h4>Warnings</h4>
+
+<p>To write a warning, an author creates a GFM blockquote whose first paragraph starts with "**Warning:**":</p>
+
+<pre>
+&gt; **Warning: This is how you write a warning.
+&gt;
+&gt; It can have multiple lines.
```

Comment:
```suggestion
&gt; It can have multiple paragraphs.
```
Alternatively, like above.